<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataBox</class>
 <widget class="QDialog" name="DataBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="nameLabel">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
       </font>
      </property>
      <property name="text">
       <string>Event Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dateLabel">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
       </font>
      </property>
      <property name="text">
       <string>Event Date:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="categoryLabel">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
       </font>
      </property>
      <property name="text">
       <string>Event Category:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="hourlabel">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
       </font>
      </property>
      <property name="text">
       <string>Event Hour(s):</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
